e2e/tests/datasets/dataset-create.spec.ts

```diff
@@ -21,8 +21,7 @@ describe('Dataset test', () => {
    * - Edit dataset. Save dataset without Export to Notebook.
    * - Delete Dataset.
    */
-  // RW-5751 Search for condition is failing in CI.
-  xtest('Can create Dataset with user-defined Cohorts', async () => {
+  test('Can create Dataset with user-defined Cohorts', async () => {
```

Comment:
[RW-5751](https://precisionmedicineinitiative.atlassian.net/browse/RW-5751) is done but no one re-enabled test.